src/clj/rems/api/schema.clj

```diff
@@ -203,13 +203,25 @@
          :field/private s/Bool
          (s/optional-key :field/previous-value) schema-base/FieldValue))
 
+(s/defschema FormData
+  {:form/internal-name s/Str
+   :form/external-title LocalizedString})
```

Comment:
I think this belongs in rems.db.form, cf. rems.db.blacklist

Reply:
I think we didn't quite finish the discussion thread about the best division of schemas so we should perhaps decide something there about this?

Reply:
Yeah #2562 is related to this. However I think since `FormData` only lives in the DB and isn't used in the API it can be moved to the DB namespace for now?

Reply:
I can do that. For this specifically, maybe the end goal is that there is no need for the `FormData` schema but we can directly use `FormTemplate` or a subset of it.